File: src/main/java/it/contrader/dto/ExperienceDTO.java
package it.contrader.dto;

import java.util.List;

import it.contrader.model.Imagen;
import it.contrader.model.User;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ExperienceDTO {
	private int idExperience;
	private int idUser;
	private int idPrincipi;
	private String commento;
	private String positivo;
	private String negativo;
	private int score;
	private List<String> secon;
	private Imagen imagen;

}

File: src/app/dto/ExperienceDTO.ts
export class ExperienceDTO{
    idUser: number; 
    idPrincipi: number;
    commento: string; 
    positivo: string; 
    negativo: string;
    score: number;
    secon: Array<string>;

    constructor(idUser: number, idPrincipi: number, commento: string, positivo: string, negativo: string, score: number,
        secon: Array<string>){
            this.commento = commento;
            this.idPrincipi = idPrincipi;
            this.idUser = idUser;
            this.negativo = negativo;
            this.positivo = positivo;
            this.score = score;
            this.secon = secon;
        }
}